tasks/main.yml

```diff
@@ -7,8 +7,9 @@
 - name: Ensure that mutually exclusive options are not set
   assert:
     that:
-      - not (galaxy_manage_clone and galaxy_manage_download)
-    fail_msg: "Variables galaxy_manage_clone and galaxy_manage_download cannot both be true."
+      - "(galaxy_manage_clone + galaxy_manage_download + galaxy_manage_existing) == 1"
```

Comment:
Whoops, this makes it impossible to run without at least one of these set (e.g. to just copy configs or manage the db). This could probably safely be:

```
      - "(galaxy_manage_clone + galaxy_manage_download + galaxy_manage_existing) <= 1"
```